src/plugins/intel_gpu/src/runtime/execution_config.cpp

```diff
@@ -250,12 +250,16 @@ void ExecutionConfig::apply_user_properties(const cldnn::device_info& info) {
     }
 
     // Enable KV-cache compression by default for non-systolic platforms
-    if (!is_set_by_user(ov::hint::kv_cache_precision) && !info.supports_immad) {
+    if (!is_set_by_user(ov::hint::kv_cache_precision) &&
+        internal_properties.find(ov::hint::kv_cache_precision.name()) == internal_properties.end() &&
+        !info.supports_immad) {
         set_property(ov::hint::kv_cache_precision(ov::element::i8));
     }
 
     // Enable dynamic quantization by default for non-systolic platforms
-    if (!is_set_by_user(ov::hint::dynamic_quantization_group_size) && !info.supports_immad) {
+    if (!is_set_by_user(ov::hint::dynamic_quantization_group_size) &&
+        internal_properties.find(ov::hint::dynamic_quantization_group_size.name()) == internal_properties.end() &&
```

Comment:
This change will disable kv_cache_compression and dynamic_group_size default configurations for non-systolic platforms entirely, because this check will never return true, as during property registration all properties are added to internal_properties with default values here.:
https://github.com/openvinotoolkit/openvino/blob/82d553e94e6eb20c3c6b24a9b1f39408d8ee51cc/src/plugins/intel_gpu/src/runtime/execution_config.cpp#L91

As a short-term solution, we can avoid using `get_property(ov::hint::dynamic_quantization_group_size)` calls at runtime for FC configuration, saving the value at model compilation stage to primitive/implementation or somewhere else
However, the proper solution would be to move these default configurations out of this function entirely and call them only once at the very beginning, either in the constructor or right after config creation